### ISSUE TITLE
css rule for th[epsection="creators_putcode"] does not find match

### ISSUE THREAD

**Issue by cziaarm** [eprints/orcid_support_advance#31]
Hello,

The bit if css that finds and hides the put-code title in the workflow is not looking for anything that it is likely to find. Is "epsection" a 3.4 attribute? There's no mention of it in core code currently: https://github.com/eprints/eprints/search?q=epsection&unscoped_q=epsection

We can hide it without epsection, with a slightly shabby 

th[id$="creators_th_N"], th[id$="editor_th_N"] {
    display: none;
}

Where N is the index of the column (yuk!)

Maybe I'm missing something regarding the epsection attr?

Cheers,

Rory

**Comment by zuphilip** [comment]
That was changed recently in 7afdc1a41fd57115716317de736efda83b0296ff.

Currently, we also see the put code field in the workflow, e.g.

![put-code-visible](https://user-images.githubusercontent.com/5199995/54111156-7b7df400-43e3-11e9-9283-f94ddc18c3a2.png)

(Note: We don't display the email field there.)

Maybe @wfyson can give a hint, whether a new ORCID Support Plugin version, ORCID Support Advance Plugin version, or EPrints version is needed for that commit to work.

**Comment by wfyson** [comment]
Hi both,

Yes, the epsection attribute is a 3.4 thing, designed to make applying CSS to the deposit workflow easier! Sorry, it was wrong of me to remove the JS that was hiding the Putcode previously and probably makes sense for both solutions to be in place.

**Comment by wfyson** [comment]
Hi @cziaarm, @zuphilip 
It looks like I got it a bit wrong earlier and that the epsection attribute is not a 3.4 thing, but is something that appears in EPrints Services versions of EPrints, but not the main public release. The other CSS rule:

`input.ep_eprint_creators_putcode`

is a 3.4 introduction however (with the aim of making this kind of task easier).

With all this in mind, I've reinstated the JS file with commit https://github.com/eprints/orcid_support_advance/commit/0ffbb5117bc9e5ff91f5670f7840ec3c14ee25f7 which together with the CSS rules should hopefully provide a more comprehensive way of dealing with the issue!
